test/goose/specs_test.clj

```diff
@@ -47,6 +47,15 @@
       #(c/perform-every tu/redis-client-opts (assoc cron-opts :cron-schedule "invalid") `tu/my-fn)
       #(c/perform-every tu/redis-client-opts (assoc cron-opts :timezone "invalid-zone-id") `tu/my-fn))
 
+    ; :batch-opts
+    (let [batch-opts {:callback-fn-sym `prn}]
+      #(c/perform-batch tu/redis-client-opts batch-opts `single-arity-fn [1 2])
+      #(c/perform-batch tu/redis-client-opts batch-opts `my-fn [["foo" "bar"]])
```

Comment:
suggestion: instead of naming `my-fn`, lets call it `absent-fn` or something like that. So that its clear why this wil lfail

Reply:
Renaming it to `unresolvable-fn`